scalafix-tests/unit/src/test/scala/scalafix/tests/core/PositionSuite.scala

```diff
@@ -0,0 +1,60 @@
+package scalafix.tests.core
+
+import org.scalatest.FunSuite
+import scala.meta.inputs.Input
+import scala.meta.inputs.Position
+import scalafix.testkit.DiffAssertions
+import scalafix.internal.util.PositionSyntax._
+
+class PositionSuite extends FunSuite with DiffAssertions {
+
+  def check(name: String, original: String, expected: String): Unit = {
+    test(name) {
+      val marker = "@@".r
```

Comment:
Can we reuse this: https://github.com/scalameta/scalameta/blob/master/tests/shared/src/test/scala/scala/meta/tests/tokenizers/BaseTokenizerCoverageSuite.scala ?

It looks really neat:
https://github.com/scalameta/scalameta/blob/master/tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerCoverageSuite.scala

Reply:
I personally dislike unicode characters since they're inconvenient to write. However, I refactored the test to use
```
  val startMarker = '→'
  val stopMarker = '←'
```
for consistency with the Scalameta tests. The extraction implementation stayed unchanged.